src/pal/src/file/path.cpp

```diff
@@ -1215,12 +1191,12 @@ SearchPathA(
     
             PathLength = pPathEnd-pPathStart;
     
-            if (PathLength+FileNameLength+1 >= MAX_LONGPATH) 
+            if (PathLength+FileNameLength+1 >= MAX_LONGPATH) //Check if we have to do anything special 
             {
                 /* The path+'/'+file length is too long.  Skip it. */
                 WARN("path component %.*s is too long, skipping it\n", 
                      (int)PathLength, pPathStart);
-                continue;
+                //continue;
```

Comment:
I have left the above check to figure out if we need to handle it specially ?
Do we still know for sure whether we are going to skip the segment, if so i will un-comment it